capacity_update.py

```diff
@@ -53,4 +53,4 @@ def capacity_update(
         update_zone(zone, parsed_target_datetime, session, update_aggregate)
 
     print("Running prettier...")
-    run_shell_command("pnpx prettier@2 --write .", cwd=ROOT_PATH)
+    run_shell_command("pnpx prettier@2 --write config/zones --cache", cwd=ROOT_PATH)
```

Comment:
`--cache` just speed things up by basically skipping files that have not been changed and limiting the scope to `config/zones` ensures only the zone config files are checked instead of everything in contrib.